packages/ra-ui-materialui/src/list/List.js

```diff
@@ -128,44 +130,47 @@ export const ListView = props => {
     const controllerProps = getListControllerProps(rest);
 
     return (
-        <div
-            className={classnames('list-page', classes.root, className)}
-            {...sanitizeRestProps(rest)}
-        >
-            <Title title={title} defaultTitle={defaultTitle} />
+        <ExporterContext.Provider value={exporter}>
```

Comment:
If we keep adding stuff to the List context (like filters, selected ids, etc), I wonder if we shouldn't use a ListContext instead... opinions?